_episodes/02-working-with-environments.md

```diff
@@ -313,15 +315,15 @@ $ conda install scikit-learn=0.22
 > [ensemble learning](https://en.wikipedia.org/wiki/Ensemble_learning), and has been widely used 
 > in real-world tasks and data science competitions like [Kaggle](https://www.kaggle.com/).
 > 
-> Activate the `machine-learning-env` you created in a previous challenge and use `pip` to install 
+> Activate the `basic-scipy-env` you created in a previous challenge and use `pip` to install 
```

Comment:
Why do you change from `machine-learning-env` to `basic-scipy-env` for this exercise? If `combo` doesn't install into the `machine-learning-env` on Windows I would prefer to open an issue and figure out why. Combo only makes sense to use with all of the deps in the `machine-learning-env` so when you install into `basic-scipy-env` these deps must be coming from `pip` when `combo` is installed. End result should be the same.

Reply:
I think we can add `numba` when creating `machine-learning-env` and it would solve our problem.

`conda create --name machine-learning-env  ipython  matplotlib  pandas  pip  python  scikit-learn numba`

Reply:
I would prefer to add `numba` to the `machine-learning-env` and then `pip install combo` into the `machine-learning-env`. Still not sure why this did't work on Windows in the first place though.

Reply:
This is already what I did.

Reply:
Right! Sorry, I see that now.